core/include/detray/propagator/navigator.hpp

```diff
@@ -22,10 +22,11 @@ namespace navigation {
 /// Navigation status flags
 enum class status {
     e_abort = -3,          ///< error ocurred, propagation will be aborted
-    e_exit = -2,           ///< navigation exited successfully
+    e_on_target = -2,      ///< navigation exited successfully
     e_unknown = -1,        ///< unknown state/not initialized
     e_towards_object = 0,  ///< move towards next object
-    e_on_target = 1,       ///< reached object
+    e_on_module = 1,       ///< reached module surface
```

Comment:
In the navigation I went with the convention now to call everything that is geometrically a surface a surface, and the two types of surfaces that the navigator can distinguish `portal` and 'module'. I don't know on which type you want to map material, but in the navigation at least, I find it clearer than before. Otherwise, I use the term 'surface' to mean two different things in the comments... If you want to use module surfaces in two different contices, you should define that outside of the navigator, I think

Reply:
But isn't module usually referring active surface?

Reply:
and navigator is supposed to find both active and passive... :/